lib/App/DuckPAN/Perl.pm

```diff
@@ -109,54 +111,73 @@ sub duckpan_install {
 		$sp =~ s/\:\:/_/g;
 
 		# special case: check for a pinned verison number
-		my $pin_version            = $ENV{$sp};
-		my $localver               = $self->get_local_version($package);
-		my $duckpan_module_version = version->parse($module->version);
-		my $duckpan_module_url     = $self->app->duckpan . 'authors/id/' . $module->distribution->pathname;
+		my $pinned_version      = $ENV{$sp};
+		my $installed_version   = $self->get_local_version($package);
+		my $latest_version      = version->parse($module->version);
+		my $duckpan_module_url  = $self->app->duckpan . 'authors/id/' . $module->distribution->pathname;
 
-		$localver ||= 1e-6 if ($pin_version); # a silly, but true, value if missing and we need to compare with pinned.
```

Comment:
I thin we still need this? Or we need a better way to handle the locally installed version. When I run `duckpan reinstall` now, I'm getting "[FATAL]  Failed to find version 9.999 of App::DuckPAN".

We set the version to `9.999` if it's a local build.